### PR TITLE
 zh: updated for ist0153

### PR DESCRIPTION
**Pull request by huan555**
Please provide a description for what this PR is for.

And to help us figure out who should review this PR, please 
put an X in all the areas that this PR affects.

- [ ] Configuration Infrastructure
- [x] Docs
- [ ] Installation
- [ ] Networking
- [ ] Performance and Scalability
- [ ] Policies and Telemetry
- [ ] Security
- [ ] Test and Release
- [ ] User Experience
- [ ] Developer Infrastructure

### CODE REVIEW

File: content/zh/docs/reference/config/analysis/ist0153/index.md [hunk 1]
```diff
@@ -0,0 +1,39 @@
+---
+title: EnvoyFilterUsesAddOperationIncorrectly
+layout: analysis-message
+owner: istio/wg-user-experience-maintainers
+test: n/a
+---
+
+当 `EnvoyFilter` 使用 `ADD` 操作且 `ApplyTo` 设置为 `ROUTE_CONFIGURATION` 或 `HTTP_ROUTE` 时，会出现此消息。
+这将导致忽略 `ADD` 操作。目前，只有 `MERGE` 操作可用于 `ROUTE_CONFIGURATION`。
```
Comment (litong01) [inline content/zh/docs/reference/config/analysis/ist0153/index.md:9]: looks like this is a bit inconsistent with the translation for 0154. can you make this similar to 0154?

这将导致`ADD` 操作被忽略

**Review by litong01 — CHANGES_REQUESTED** [review]

**Comment by litong01** [comment]
also missing CLA. please add your signature.

File: content/zh/docs/reference/config/analysis/ist0153/index.md [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+owner: istio/wg-user-experience-maintainers
+test: n/a
+---
+
+当 `EnvoyFilter` 使用 `ADD` 操作且 `ApplyTo` 设置为 `ROUTE_CONFIGURATION` 或 `HTTP_ROUTE` 时，会出现此消息。
+这将导致 `ADD` 操作被忽略。目前，只有 `MERGE` 操作可用于 `ROUTE_CONFIGURATION`。
+
+## 例如{#example}
+
+以下示例中，如果一个 `EnvoyFilter` 附带有 `ADD` 补丁操作 ，该 `EnvoyFilter` 将被忽略：
```
Comment (windsonsea) [inline content/zh/docs/reference/config/analysis/ist0153/index.md:13]:
```suggestion
以下示例中，如果一个 `EnvoyFilter` 附带有 `ADD` 补丁操作，该 `EnvoyFilter` 将被忽略：
```

**Comment by windsonsea** [comment]
@huan555 Could you move this forward? 19 days passed.

**Comment by windsonsea** [comment]
Hi @huan555 , could you take a look over the comment? If no any response for certain days, this PR will be closed. Thanks.

**Comment by huan555** [comment]
@windsonsea sorry for late, the change have been commited.

**Review by windsonsea — APPROVED** [review]

**Review by zirain — APPROVED** [review]

**Pull request merged**